api/WorkerNavigator.json

```diff
@@ -6,10 +6,10 @@
         "spec_url": "https://html.spec.whatwg.org/multipage/workers.html#the-workernavigator-object",
         "support": {
           "chrome": {
-            "version_added": "1"
+            "version_added": "31"
```

Comment:
This does not seem plausible to me, and seems like it's probably a test issue. I've sent https://github.com/foolip/mdn-bcd-collector/pull/1194 to use an instance.

I would expect `api.WorkerNavigator` data to match `api.Worker`, which it didn't before but it was much closer.

Reply:
I'll go ahead and revert this for now as I find the data questionable, though I believe Chrome 1 is also incorrect (considering worker support was added in Chrome 4).  Something to revisit later, though!